logstash-core/src/main/java/org/logstash/plugins/inputs/Stdin.java

```diff
@@ -24,53 +21,54 @@
 import java.util.Collections;
 import java.util.Map;
 import java.util.concurrent.CountDownLatch;
-import java.util.concurrent.atomic.LongAdder;
 import java.util.function.Consumer;
 
-@LogstashPlugin(name = "java-stdin")
+@LogstashPlugin(name = "java_stdin")
 public class Stdin implements Input, Consumer<Map<String, Object>> {
 
-    private static final Logger logger = LogManager.getLogger(Stdin.class);
+    private final Logger LOGGER;
```

Comment:
The all-caps variable name makes me think that this would be static. Since it is instance-specific, should we go back to calling it `logger`?